J/JSBSim/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+using BinaryBuilder
+
+name = "JSBSim"
+version = v"1.1.6"
+julia_version = v"1.5.3"
+
+# Collection of sources required to build JSBSim
+sources = [
+    GitSource("https://github.com/bcoconni/jsbsim.git",
+              "a31ecb8727361aa74dee4ea7158c944848f78fb6"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/jsbsim
+mkdir build && cd build
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+    FLAGS+=(-DCMAKE_CXXFLAGS_RELEASE="-D_POSIX_C_SOURCE")
+fi
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBUILD_DOCS=OFF \
+    -DBUILD_PYTHON_MODULE=OFF \
+    -DBUILD_JULIA_PACKAGE=ON \
+    -DJulia_PREFIX="$prefix" \
```

Comment:
CMake still doesn't seem to like it despite this line.  I wonder if @fingolfin can give us some hints (and avoid me descending into CMake madness)